node/heartbeat/monitor.go

```diff
@@ -160,7 +167,9 @@ func (m *Monitor) loadRestOfPubKeysFromStorage() error {
 	for _, peer := range peersSlice {
 		pubKey := string(peer)
 		_, ok := m.heartbeatMessages[pubKey]
+		log.Info("loaded heartbeat for pubkey from storage", "pubkey", peer)
```

Comment:
is the info needed here? Could it be just on debug?

Reply:
sorry, just used it for debugging. will remove it

Reply:
ok